client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -9,28 +9,123 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+import { CompactCard } from '@automattic/components';
 import DisconnectSurvey from './disconnect-survey';
 import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
+import Gridicon from 'components/gridicon';
 import Main from 'components/main';
 import NavigationLink from 'components/wizard/navigation-link';
 import redirectNonJetpack from 'my-sites/site-settings/redirect-non-jetpack';
 import Troubleshoot from './troubleshoot';
-import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { getSelectedSiteSlug, getSelectedSite } from 'state/ui/selectors';
+import { localizeUrl } from 'lib/i18n-utils';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const DisconnectSite = ( { reason, siteSlug, translate } ) => {
-	const confirmHref = '/settings/disconnect-site/confirm/' + siteSlug;
+const DisconnectSite = ( {
+	reason,
+	type,
+	siteSlug,
+	site: { name: siteName, URL: siteUrl },
+	translate,
+} ) => {
+	const confirmHref = `/settings/disconnect-site/confirm/${ siteSlug }?type=down`;
 
 	let backHref = '/settings/manage-connection/' + siteSlug;
```

Comment:
bug: the Back buttons on both old and new pages always take you back to Manage Connection (don’t worry about the reason branch below, it looks like a red herring), but if you used the CTA, that’s not necessarily where you came from

Reply:
should i make this onClick history.back + preventDefault and keep the href as a fallback, or is there a way to get the previous URL out of the router and set href to that?

Reply:
I would be pretty cool if we can actually make it go back in history rather than dropping them in the Manage Connection page. Even though they won't be able to do much since the connection is broken, it would be nice that all the `<- Back` links would behave in an expectable way.

Reply:
A simple
```js
import page from 'page';
page.back();
```
should be enough to go back.

Or `window.history.back()`. The difference is that `page.back()` will never navigate out of Calypso (i.e., to another page that was previously opened in the same tab.

If you want to construct a full `<a href="..."/>` with a valid link, then the `getPreviousRoute( state )` Redux selector will return the previous URL.

All of the above will work only if there is some navigation history to work with. If you got to the page by opening a direct link (from email, from chat, ...), clicking on the "back" arrow will do nothing.

Reply:
wow there are more options than i realised, and we could combine them in several ways, plus there’s also the question of whether this NavigationLink should go back through history (enabling the browser forward button) or push forward to a new history entry… does calypso have a convention i can defer to here?

Reply:
![child laughing then crying](https://user-images.githubusercontent.com/465303/84477282-b94a8600-acd2-11ea-88b3-7605a08fe753.png)